Homework_1_method_strings.py

```diff
@@ -0,0 +1,139 @@
+# Method Strings Homework 1
+
+# String
+break_thing = "*"*100
+
+#Slicing() eturn a range of characters by using the slice syntax.
+
+sentence = "Hello, how are you? my name is Daniela and yours?"
+print(sentence[0:5])
+print(sentence[20:])
+
+print(break_thing)
+
+#len() get the length of a string
+
+names = ["Rodrigo", "Daniela", "Cody", "Bernardo"]
+if names[0] == "Rodrigo":
+    print(len(names[2]))
+else:
+    print("error")
+
+print(break_thing)
+
+#sort
+
+data = ["17809", "20203", "14590", "167777", "29402"]
+
+def sort_postal_codes(codes):
+    for number in codes:
+         if number[0] == "1":
+            print(number)
+
+sort_postal_codes(data)
+
+print(break_thing)
+
+#lower and upper() returns the string in lower/upper case.
+
+a = "ME LLAMO"
+b = "daniela"
+print a.lower(), b.upper()
+
+print(break_thing)
+
+#strip() removes any whitespace from the beginning or the end.
+
+a = "           I think Cody is very cute            "
+print(a.strip())
+
+print(break_thing)
+
+#replace() eplaces a string with another string.
+
+fruits_with_a = "Opple, Opricot"
+fruits_with_b = "Eanana, Elueberries"
+print(fruits_with_a.replace("O", "A"))
+print(fruits_with_b.replace("E", "B"))
+
+print(break_thing)
+
+#split() returns a list of strings after breaking the given string by the specified separator.
+
+phrase = "Like, Apples?"
+print phrase.split(",")
+
+print(break_thing)
+
+#capitalize() converts the first character of a string to uppercase letter.
+
+name = "daniela"
+last_name = "gonzalez"
+second_last_name = "barron"
+print(name.capitalize()) + last_name.capitalize() + second_last_name.capitalize()
```

Comment:
```suggestion
print(name.capitalize() + " " + last_name.capitalize() + " " + second_last_name.capitalize())
```
print() is a function. For this reason, it's best to have everything you want to print inside of it. Also, the + " " +'s are to add spaces within the names